iree/hal/cuda/dynamic_symbols.c

```diff
@@ -20,6 +20,14 @@ static const char* kCUDALoaderSearchNames[] = {
 #endif
 };
 
+static const char* kCUPTILoaderSearchNames[] = {
+#if defined(IREE_PLATFORM_WINDOWS)
+    // TODO (kooljblack): determine windows cupti dll
```

Comment:
zero length arrays don't work
"cupti64_2021.2.0.dll" is what you want on windows